### ISSUE TITLE
[0.7] Error in solidity::frontend::Visibility

### ISSUE THREAD

**Issue by hajduakos** [SRI-CSL/solidity#153]
Many contracts fail with
```
> Error while running compiler, details:
> Warning: This is a pre-release compiler version, please do not use it in production.
> 
> 
> ======= Converting to Boogie IVL =======
> 
> ======= test/solc-verify/arrays/ArrayInit.sol =======
> solc-verify internal exception: 
> Details:
> /home/user/workspace/solidity/libsolidity/ast/AST.cpp(294): Throw in function virtual solidity::frontend::Visibility solidity::frontend::FunctionDefinition::defaultVisibility() const
> Dynamic exception type: boost::exception_detail::clone_impl<solidity::langutil::InternalCompilerError>
> std::exception::what: 
> [solidity::util::tag_comment*] = 
> 
```
For example `arrays/ArrayInit.sol`

**Comment by dddejan** [comment]
`arrays/ArrayInit.sol` is fixed in a8c7207671e1e488ee8b0d3032ddcd6fb3ff885b.

**Comment by hajduakos** [comment]
As of 60575f7, I do not see this error anymore, so I close this one
